templates/releng/netboot.html

```diff
@@ -46,18 +46,18 @@ <h3>Download</h3>
 <h4>BIOS</h4>
 
 <ul>
-<li><a href="{% static "netboot/ipxe.lkrn" %}">ipxe.lkrn</a> (<a href="{% static "netboot/ipxe.lkrn.sig" %}">PGP signature</a>) - Linux kernel-like image that can be started with any Linux bootloader</li>
-<li><a href="{% static "netboot/ipxe.pxe" %}">ipxe.pxe</a> (<a href="{% static "netboot/ipxe.pxe.sig" %}">PGP signature</a>) - PXE image for chainloading from a PXE environment</li>
+<li><a href="{% static "netboot/ipxe-arch.lkrn" %}">ipxe-arch.lkrn</a> (<a href="{% static "netboot/ipxe-arch.lkrn.sig" %}">PGP signature</a>) - Linux kernel-like image that can be started with any Linux bootloader</li>
+<li><a href="{% static "netboot/ipxe-arch.pxe" %}">ipxe-arch.pxe</a> (<a href="{% static "netboot/ipxe-arch.pxe.sig" %}">PGP signature</a>) - PXE image for chainloading from a PXE environment</li>
 </ul>
 
 <h4>UEFI</h4>
 
 <ul>
-<li><a href="{% static "netboot/ipxe.efi" %}">ipxe.efi</a> (<a href="{% static "netboot/ipxe.efi.sig" %}">PGP signature</a>) - x86_64 UEFI executable</li>
+<li><a href="{% static "netboot/ipxe-arch.efi" %}">ipxe-arch.efi</a> (<a href="{% static "netboot/ipxe-arch.efi.sig" %}">PGP signature</a>) - x86_64 UEFI executable</li>
 </ul>
 
 <p>You can build your own iPXE images compatible with netboot.
-For details, check out the <a href="https://aur.archlinux.org/packages/ipxe-netboot">ipxe-netboot AUR package</a>.</p>
+For details, check out the <a href="https://archlinux.org/packages/community/x86_64/ipxe/">ipxe package</a>.</p>
```

Comment:
ideally we would not hardcode the url but obtain it from a pkg istance with  `{{ pkg.get_full_url }}`

So `netboot_info` has to be expanded and obtain the package and pass it to the template.